graphql-schemas/schema.yml

```diff
@@ -2386,6 +2386,11 @@ confs:
   fields:
   - { name: slack, type: boolean }
 
+- name: IntegrationSpecShardSpecOverride_v1
+  fields:
+  - { name: account, type: AWSAccount_v1 }
+  - { name: shard, type: string }
```

Comment:
what is the purpose of those fields? from a high level perspective, both of them seem to be a specifier for the account the overwrite is for

isn't the imageRef missing?

Reply:
Right, this branch is outdated. Will push an update soon